src/templates/upload-dropdown-menu.html

```diff
@@ -14,6 +14,20 @@
       </button>
     {% endif %}
 
+    {# INSERT IN TEXT in edit mode for videos #}
+    {% if ext matches '/(mpeg|mpg|mpe|mp4|ogv|webm|avi)$/i' %}
```

Comment:
I suggest limiting to mp4|ogv|webm|hevc|av1

See the Sub-features tab of `video` element here: https://caniuse.com/?search=video

We need to stick to what the browser actually supports in `video`, not every video format under the sun ;)

Reply:
Your're right, but then this list boils down to `mp4|ogv|webm` which are the actual possible file extensions. Might also be the right call to include `mkv` which is another container format (and ancestor to webm) supported by all major browsers. What really hits us here is that we are matching file extensions while what matters from a browser's point of view are video codecs (hvec, av1, vp9, etc.). So while we don't want to include only video codecs supported by browsers, those may be contained in a variety of widely-used container formats.

Reply:
Maybe an evolution could be to store the MIME type upon upload.